src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -415,9 +415,9 @@ def storage_blob_upload_batch(cmd, client, source, destination, pattern=None,  #
                               maxsize_condition=None, max_connections=2, lease_id=None, progress_callback=None,
                               if_modified_since=None, if_unmodified_since=None, if_match=None,
                               if_none_match=None, timeout=None, dryrun=False, socket_timeout=None, **kwargs):
-    def _create_return_result(blob_content_settings, upload_result=None):
+    def _create_return_result(blob_content_settings, upload_result=None, blob_client=None):
```

Comment:
`blob_client` is a required argument, right? We can't accept a `None` as default...

Reply:
Got it